### PR TITLE
Added new shim for uwp console. 

### PR DESCRIPTION
**Pull request by sognerubov**
Added new shim for uwp console.
Removed the constriction for the number of args.
Replaced all the returns in the main with TerminateProcess.
Replaced using the first arg with GetModuleFileName.

### CODE REVIEW

**Comment by sognerubov** [comment]
@nattress please review

**Comment by zamont** [comment]
I'll defer to @nattress on this review

File: src/uwp/Microsoft.Build.Net.CoreRuntimeTask/WireUpCoreRuntime.cs [hunk 1]
```diff
@@ -145,25 +146,35 @@ internal int InternalExecute()
...
                 if (isTargetRuntimeManaged)
                 {
                     // 1. For Managed apps, move <app.exe> to [AppEntryPointDir]\<app.exe> and copy UWPShim.exe to package root as <app.exe>
-                    entryPointExecutables = doc.Descendants(ns + "Applications").Descendants(ns + "Application").Where(x => x.Attribute("Executable") != null).Select(x => x.Attribute("Executable"));
+                    var entryPointExecutables = doc.
+                                                Descendants(ns + "Applications").
+                                                Descendants(ns + "Application").
+                                                Where(x => x.Attribute("Executable") != null).
+                                                Select(x => new { Executable = x.Attribute("Executable").Value,
+                                                                  ShimFileName = "console".Equals(x.Attribute("desktop4:Subsystem").Value, StringComparison.OrdinalIgnoreCase) ? consoleUwpShimLocation : windowsUwpShimLocation});
```
Comment (sognerubov) [inline src/uwp/Microsoft.Build.Net.CoreRuntimeTask/WireUpCoreRuntime.cs]: is desktop4:Subsystem is fine?

Comment (sognerubov) [inline src/uwp/Microsoft.Build.Net.CoreRuntimeTask/WireUpCoreRuntime.cs]: I need to use?. here

**Pull request merged**